### PR TITLE
PMM-8278 add SHA256 hash to oneline script

### PR DESCRIPTION
**Pull request by nikita-b**
We need to merge checksum before this PR: https://github.com/percona/pmm/pull/756

### CODE REVIEW

**Comment by nikita-b** [comment]
@EvgeniyPatlan We have other PR with checksum: https://github.com/percona/pmm/pull/756

**Comment by EvgeniyPatlan** [comment]
@nikita-b why it can't be done in 1 PR? So it could be tested together.

File: docs/setting-up/server/easy-install.md [hunk 1]
```diff
@@ -4,10 +4,20 @@
     - This is a [technical preview] and is subject to change.
     - Download and check `get-pmm2.sh` before running it to make sure you know what it does.
 
+### Linux:
 ```sh
-curl -fsSL https://raw.githubusercontent.com/percona/pmm/PMM-2.0/get-pmm.sh -o get-pmm2.sh && \
-chmod +x get-pmm2.sh && \
-./get-pmm2.sh
+curl -fsSL -O https://raw.githubusercontent.com/percona/pmm/PMM-2.0/get-pmm.sh -O https://raw.githubusercontent.com/percona/pmm/PMM-2.0/.sha256-oneline && \
+sha256sum .sha256-oneline -c && \
```
Comment (atymchuk) [inline docs/setting-up/server/easy-install.md:10]: `sha256sum` is not installed by default on MacOS, while `shasum` is. Can we rewrite this to use `shasum`, which seem to be able to handle them all with a parameter `-a ...`?

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: docs/setting-up/server/easy-install.md (hunk 1 above)
Comment (nikita-b) [reply docs/setting-up/server/easy-install.md:10]: We also don't have `shasum` on Linux (CentOs) by default :(
It's part of `perl-Digest-SHA` package.

I think I can run different tools depends on OS but the script will be more complicated.

Comment (atymchuk) [reply docs/setting-up/server/easy-install.md:10]: Ok then, let's leave it as is )

**Review by atymchuk — APPROVED** [review]

**Comment by nikita-b** [comment]
> @nikita-b why it can't be done in 1 PR? So it could be tested together.

Our get-pmm script in PMM repo:
https://github.com/percona/pmm/blob/PMM-2.0/get-pmm.sh

I think It'll be strange and inconvenient if the checksum and the file are in different repositories. Maybe we can move the script to docs repo...I'm not sure. @atymchuk what do you think?

**Comment by EvgeniyPatlan** [comment]
> > @nikita-b why it can't be done in 1 PR? So it could be tested together.
> 
> Our get-pmm script in PMM repo:
> https://github.com/percona/pmm/blob/PMM-2.0/get-pmm.sh
> 
> I think It'll be strange and inconvenient if the checksum and the file are in different repositories. Maybe we can move the script to docs repo...I'm not sure. @atymchuk what do you think?

Why we couldn't generate hash on the fly?

**Comment by atymchuk** [comment]
> > @nikita-b why it can't be done in 1 PR? So it could be tested together.
> 
> Our get-pmm script in PMM repo:
> https://github.com/percona/pmm/blob/PMM-2.0/get-pmm.sh
> 
> I think It'll be strange and inconvenient if the checksum and the file are in different repositories. Maybe we can move the script to docs repo...I'm not sure. @atymchuk what do you think?

Does sound like making sense...

**Comment by atymchuk** [comment]
> > > @nikita-b why it can't be done in 1 PR? So it could be tested together.
> > 
> > 
> > Our get-pmm script in PMM repo:
> > https://github.com/percona/pmm/blob/PMM-2.0/get-pmm.sh
> > I think It'll be strange and inconvenient if the checksum and the file are in different repositories. Maybe we can move the script to docs repo...I'm not sure. @atymchuk what do you think?
> 
> Why we couldn't generate hash on the fly?

What we want is the opposite - a verifiable hash needs to be committed to the repo.

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by PaulJacobs-percona — APPROVED** [review]

**Pull request merged**